astroquery/simbad/core.py

```diff
@@ -698,7 +698,7 @@ def query_objects(self, object_names, *, wildcard=False, criteria=None,
 
     @deprecated_renamed_argument(["equinox", "epoch", "cache"],
                                  new_name=[None]*3,
-                                 alternative=["astropy.coordinates.SkyCoord"]*3,
+                                 alternative=["astropy.coordinates.SkyCoord"]*2 + [None],
```

Comment:
Unrelated to this PR, but looking at this line now, these are not really alternatives for the removed arguments, thus I would remove it altogether.

(e.g. I find this message more confusing than helpful:

```
WARNING: AstropyDeprecationWarning: "equinox" was deprecated in version 0.4.8 and will be removed in a future version. 
        Use ['astropy.coordinates.SkyCoord', 'astropy.coordinates.SkyCoord', None] instead. [warnings]
```